shell/assets/styles/global/_columns.scss

```diff
@@ -1,6 +1,6 @@
 @use "sass:math";
 
-@import "~shell/assets/styles/base/functions";
```

Comment:
What does this do? 

From what I read it was to prevent additional css files from being generated but I think we inline all of our css.

Reply:
To be fair I forgot what was the initial error since the picked commit is 3 months old 😅 
There was an issue importing though, so I made it standard.